test/coercion_test.rb

```diff
@@ -39,7 +39,7 @@ class VirtusCoercionTest < MiniTest::Spec
     representer! do
       include Representable::Coercion
 
-      property :length, :type => Float,
+      property :length, :type => Representable::Coercion::Types::Coercible::Float,
```

Comment:
@apotonick because `representer!` creates an anonymous module, we need to fully namespace `Types` here.